pkgs/development/haskell-modules/configuration-ghc-8.0.x.nix

```diff
@@ -66,12 +69,106 @@ self: super: {
   kan-extensions = self.kan-extensions_5_0_1;
 
   # Earlier versions don't support kan-extensions 5.x.
-  lens = self.lens_4_13_2_1;
+  # released as 4.14, upstream lens with latest GHC 8 fixes
+  lens           = haskell.lib.overrideCabal super.lens_4_13_2_1 (oldAttrs: {
+    src = pkgs.fetchgit {
```

Comment:
lens 4.14 is available in `master` now. Generally speaking, the latest version of every package shows up in `master` within a few days of hitting Hackage.